app/experimenter/experiments/tests/api/v4/test_serializers.py

```diff
@@ -116,8 +115,8 @@ def test_serializer_outputs_expected_schema_with_nameSpace_bucket(self):
             {
                 "id": normandy_slug,
                 "filter_expression": "env.version|versionCompare('80.0') >= 0",
-                "targeting": "localeLanguageCode == 'en' && region == 'US'"
-                " && browserSettings.update.channel == 'release'",
+                "targeting": '[userId, "experimenter-normandy-slug"]|bucketSample(0, 100, 10000) '
+                "&& localeLanguageCode == 'en' && region == 'US' && browserSettings.update.channel == 'release'",
```

Comment:
I think flake8 is failing the build on these 2 lines (E501 line too long):

```sh
/usr/local/lib/python3.8/site-packages/django_filters/rest_framework/backends.py:147: UserWarning: <class 'experimenter.experiments.api.v2.views.ExperimentCSVListView'> is not compatible with schema generation
  warnings.warn(
./experimenter/experiments/tests/api/v4/test_serializers.py:118:91: E501 line too long (98 > 90 characters)
./experimenter/experiments/tests/api/v4/test_serializers.py:119:91: E501 line too long (113 > 90 characters)
parallel: This job failed:
flake8 .
```

Reply:
asgasghaslkghaslgjhasljggh 😫 , lemme fix this... thanks @pdehaan